core/modules/cart/helpers/createMethodsData.ts

```diff
@@ -0,0 +1,44 @@
+import ShippingDetails from '@vue-storefront/core/modules/checkout/types/ShippingDetails'
+import { ShippingMethodsData } from '@vue-storefront/core/modules/cart/types/ShippingMethodsData'
+
+const getShipping = (shippingMethods: any[]) => {
+  const shipping = shippingMethods && Array.isArray(shippingMethods) ? shippingMethods.find(item => item.default && !item.offline) : null
```

Comment:
we know it's an array, we provided default argument value, find returns null 
```suggestion
  const onlineShippingMethods = shippingMethods.filter(shippingMethod => !shippingMethod.offline)
  if (!onlineShippingMethods.length) return
  return onlineShippingMethods.find(shippingMethod => !!shippingMethod.default) || onlineShippingMethods[0]
```

and rest of this method can be removed then